crates/containerd-shim-wasm/src/sandbox/shim.rs

```diff
@@ -746,10 +746,10 @@ impl<T: Instance + Send + Sync> Local<T> {
         );
         InstanceData {
             instance: None,
-            base: Some(Nop::new(id, None)),
+            base: Some(Nop::new(id, None).unwrap()),
```

Comment:
this is safe since `Nop::new()` always returns `Ok`.  might be work a comment incase things change in future?